cmd/shfmt/main.go

```diff
@@ -26,6 +26,7 @@ var (
 	simple = flag.Bool("s", false, "")
 	find   = flag.Bool("f", false, "")
 	diff   = flag.Bool("d", false, "")
+	color  = flag.Bool("c", false, "")
```

Comment:
sorry, but no flag - this should be on by default on a terminal, and it can be turned off with TERM=dumb

Reply:
Oh, sorry I missed that comment. I fixed it.